cypress/e2e/buy-shirt.cy.ts

```diff
@@ -20,9 +22,12 @@ describe("Buy a t-shirt", () => {
     menuContentPage.goToTShirtMenu();
     productListPage.addToCart();
     shoppingToCart.proceedToCheckout();
-    loginPage.logginPageSignIn();
+    signIn.signInPage("aperdomobo@gmail.com", "WorkshopProtractor");
```

Comment:
It will be better if you rename the object as `signInPage`, and then the method just `signIn`